### DISCUSSION TITLE
ALT+Down shortcut rebind

### DISCUSSION THREAD

**Discussion by heit0r** [aclap-dev/video-downloadhelper#206] [Help and Q&A]
### Your question

Is there a way to erase or rebind the Alt+Down shortcut?

### Addon & CoApp version and details

version: 8.2.0.23
channel: stable
build date: 2024-02-14
build options: linuxlic: false, noyt: false, browser: firefox
lang: pt-BR
license: unneeded
platform: x86-64 linux
UA: Mozilla/5.0 (X11; Linux x86_64; rv:122.0) Gecko/20100101 Firefox/122.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/home/mojo/.local/share/vdhcoapp/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "linux",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/home/mojo",
  "converterBinary": "/home/mojo/.local/share/vdhcoapp/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360"
}

**Comment by GrampaWildWilly** [comment]
![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/0c03f370-f011-4f13-b34f-f942f57a150f)
